src/lib/data.json

```diff
@@ -0,0 +1,226 @@
+          "head": [0, 13600, 51800, 82500, 157500, 200000, 500000],
+          "ss": 128400,
+          "budget": 6300000000000,
+          "budgetPercents": [0.1678, 0.1646, 0.1577, 0.1103, 0.0868, 0.0848, 0.0685, 0.0326, 0.0209, 0.0205, 0.0184, 0.0134, 0.0122, 0.0121, 0.009, 0.006, 0.0045, 0.0029, 0.0069],
+          "standardDeduction": {
+            "single": 12000,
+            "separate": 12000,
+            "together": 24000,
+            "head": 18000
+          }
+      },
+
+      "2019": {
+          "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+          "single": [0, 9700, 39475, 84200, 160725, 204100, 510300],
+          "seperate": [0, 9700, 39475, 84200, 160725, 204100, 306175],
+          "together": [0, 19400, 78950, 168400, 321450, 408200, 612350],
+          "head": [0, 13850, 52850, 84200, 160700, 204100, 510300],
+          "ss": 132900,
+          "budget": 6600000000000,
+          "budgetPercents": [0.1756, 0.1651, 0.1597,0.1098,0.0859,0.0884,0.0608,0.0328,0.0195,0.0235,0.0181,0.0092,0.0110,0.0124,0.0079,0.0059,0.0061,0.0037,0.0048],
+          "standardDeduction": {
+            "single": 12200,
+            "seperate": 12200,
+            "together": 24400,
+            "head": 18350
+          }
+        },
+
+        "2020" : {
+            "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+            "single": [0, 9875, 40125, 85525, 163300, 207350, 518400],
+            "seperate": [0, 9875, 40125, 85525, 163300, 207350, 311025],
+            "together": [0, 19750, 80250, 171050, 326600, 414700, 622050],
+            "head": [0, 14100, 53700, 85500, 163300, 207350, 518400],
+            "ss": 137700,
+            "budgetPercents": [0.1452, 0.1261, 0.1203, 0.1038, 0.1592, 0.0604, 0.0649, 0.0269, 0.0102, 0.0313, 0.0215, 0.0201, 0.0084, 0.0101, 0.0714, 0.0045, 0.0045, 0.0034, 0.0077],
+            "standardDeduction": {
+              "single": 12400,
+              "seperate": 12400,
+              "together": 24800,
+              "head": 18650
+            }
+          },
+
+        "2021" : {
+            "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+            "single": [0, 9950, 40525, 86375, 164925, 209425, 523600],
+            "seperate": [0, 9950, 40525, 86375, 164925, 209425, 314150],
+            "together": [0, 19900, 81050, 172750, 329850, 418850, 628301],
+            "head": [0, 14200, 54200, 86350, 164900, 209400, 523600],
+            "ss": 142800,
+            "budgetPercents": [0.1374, 0.1178, 0.1111, 0.0998, 0.1987, 0.0576, 0.0696, 0.0256, 0.0195, 0.0505, 0.0218, 0.0125, 0.0078, 0.0087, 0.0387, 0.0041, 0.0036, 0.0018, 0.0131],
+            "standardDeduction": {
+              "single": 12550,
+              "seperate": 12550,
+              "together": 25100,
+              "head": 18800
+            }
+        },
+        "2022" : {
+            "percents": [0.1,0.12,0.22,0.24,0.32,0.35,0.37
+            ],
```

Comment:
Can you move this bracket up to the next line?
Also can you add spaces between entries on all of the lists for this year and any other years? That would make it easier for me to validate the data.